spec/datetimeformat.html

```diff
@@ -186,6 +186,19 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
             1. Let _rangePatterns_ be _bestFormat_.[[rangePatterns]].
         1. Set _dateTimeFormat_.[[Pattern]] to _pattern_.
         1. Set _dateTimeFormat_.[[RangePatterns]] to _rangePatterns_.
+        1. Let _nf1Options_ be OrdinaryObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_nf1Options_, *"useGrouping"*, *false*).
+        1. Set _dateTimeFormat_.[[NumberFormat1]] to ? Construct(%NumberFormat%, &laquo; _r_.[[locale]], _nf1Options_ &raquo;).
```

Comment:
I think I'd prefer to use `dateTimeFormat.[[Locale]]` here, because that matches the surrounding code a bit better.
```suggestion
        1. Set _dateTimeFormat_.[[NumberFormat1]] to ? Construct(%NumberFormat%, &laquo; _dateTimeFormat_.[[Locale]], _nf1Options_ &raquo;).
```